apps/state/config/config.exs

```diff
@@ -37,8 +37,10 @@ config :state, :shape,
     "811_0007" => -1,
     # Green-B (North Station)
     "811_0008" => -1,
-    # Green-B (North Station)
+    # Green-B (North Station),
     "811_0009" => -1,
+    # Green-B (North Station),
```

Comment:
The comma in the comment isn't necessary, per the convention in the rest of this file.